### PR TITLE
Bring paginator in line with wires

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

**Comment by MattPutnam** [comment]
screen shot 2018-05-09 at 5 20 18 pm

Do we care about this?

File: src/components/AuthContainer.js [hunk 1]
```diff
@@ -54,8 +54,8 @@ export default class AuthContainer extends Component {
     const { isSignedIn, isShowingNotRegisteredModal } = this.state
     return h(Fragment, [
       div({ id: 'signInButton', style: { display: isSignedIn ? 'none' : 'block' } }),
-      isShowingNotRegisteredModal ? this.renderNotRegisteredModal() : null,
-      isSignedIn ? children : null
+      isShowingNotRegisteredModal && this.renderNotRegisteredModal(),
+      isSignedIn && children
```
Comment (panentheos) [inline src/components/AuthContainer.js:58]: TOL: I support this refactor, but broad changes like this should generally go in their own PR to reduce noise.

File: src/components/table.js [hunk 2]
```diff
@@ -24,26 +31,70 @@ const paginator = function(props) {
...
+              [icon('angle-double left', { size: 12 })]
+            ),
+
+            paginatorButton(
+              _.merge({ disabled: !hasPreviousPage, style: { marginRight: '1rem' } },
+                getPageItemProps({ pageValue: previousPage, onPageChange: setPageNumber })),
+              [icon('angle left', { size: 12 })]
+            ),
+
+            h(Fragment, _.map(pages, num => paginatorButton(
```
Comment (panentheos) [inline src/components/table.js]: TOL: As a rule of thumb, I think Fragments should be reserved for cases where the contents are statically known, and dynamic content should use keys. In some cases, keys can help React re-render more efficiently, e.g. if an element is removed or inserted near the beginning of a long list.

**Review by panentheos — APPROVED** [review]
Good library choice, I like how it provides some utility with no assumptions about markup.

File: src/components/AuthContainer.js (hunk 1 above)
Comment (zarsky-broad) [reply src/components/AuthContainer.js:58]: You are completely correct, I'm often too impatient for that and I need to work on it.
